### PR TITLE
Display assessment results

### PR DESCRIPTION
**Pull request by lowjoel**

### CODE REVIEW

**Comment by lowjoel** [comment]
@allenwq I think we need to go fix Rubocop lint warnings soon...

**Comment by lowjoel** [comment]
@allenwq I wanted to override link_to_user in courses to support CourseUser but that would cause tests that randomly fail...

File: app/helpers/course/controller_helper.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+module Course::ControllerHelper
+  # Formats the given +CourseUser+ as a user-visible string.
+  #
+  # @param [CourseUser] user The User to display.
+  # @return [String] The user-visible string to represent the User, suitable for rendering as
+  #   output.
+  def display_course_user(user)
+    user.name
```
Comment (allenwq) [inline app/helpers/course/controller_helper.rb]: @lowjoel Shall we display `course_user.user.name` here ?  Thus all the names will be consistent.  Or just leave it here and change in future when needed ?

Comment (lowjoel) [inline app/helpers/course/controller_helper.rb]: No. The Course User has a name that is to be displayed within the course.

Comment (allenwq) [inline app/helpers/course/controller_helper.rb]: Yah, but things like creator, we are actually displaying the user name in course

Comment (lowjoel) [inline app/helpers/course/controller_helper.rb]: Then those need to be fixed.

Comment (allenwq) [inline app/helpers/course/controller_helper.rb]: I think it's everywhere.

Comment (lowjoel) [inline app/helpers/course/controller_helper.rb]: I don't see any places where it's being used by grepping?

Comment (lowjoel) [inline app/helpers/course/controller_helper.rb]: Regardless, this is the correct behaviour.

Comment (allenwq) [inline app/helpers/course/controller_helper.rb]: Like announcement creator, forum post creator

Comment (allenwq) [inline app/helpers/course/controller_helper.rb]: Sure

Comment (lowjoel) [inline app/helpers/course/controller_helper.rb]: I only saw this in announcements, forum needs to be fixed

Comment (allenwq) [inline app/helpers/course/controller_helper.rb]: OK !

**Pull request merged**